perception_dataset/rosbag2/autoware_msgs.py

```diff
@@ -129,18 +140,29 @@ def get_category_name(classification: List[ObjectClassification]) -> str:
 
     scene_annotation_list: List[Dict[str, Any]] = []
     for obj in msg.objects:
-        obj: Union[DetectedObject, TrackedObject]
+        obj: Union[
+            AutowareAutoDetectedObject,
+            AutowareAutoTrackedObject,
+            AutowareDetectedObject,
+            AutowareTrackedObject,
+        ]
         pose = obj.kinematics.pose_with_covariance.pose
 
-        if isinstance(obj, DetectedObject):
+        print(
+            AutowareAutoDetectedObject,
+            AutowareAutoTrackedObject,
+            AutowareDetectedObject,
+            AutowareTrackedObject,
+        )
```

Comment:
[Q] What is the purpose of this `print`? Can we remove it?

```suggestion
```

Reply:
Seems I forgot to remove debugging print. Fixed: https://github.com/tier4/tier4_perception_dataset/pull/110/commits/9ffbd3aa4be03a3619a9bd17acd74b67b22ce9a3